cypress/e2e/shared/dashboardsIndex.test.ts

```diff
@@ -252,6 +253,183 @@ describe('Dashboards', () => {
     )
   })
 
+  describe('cloning', () => {
+    const localDashName = 'Happy Dashboard'
+    const labelName = 'bark'
+    const secondLabelName = 'prrr'
+    beforeEach(() => {
+      cy.writeLPDataFromFile({
+        filename: 'data/wumpus01.lp',
+        offset: '20m',
+        stagger: '1m',
+      }).should('equal', 'success')
+
+      return cy.get<Organization>('@org').then(({id}: Organization) =>
+        cy.createDashboard(id, localDashName).then(({body}) =>
+          cy.createAndAddLabel('dashboards', id, body.id, labelName).then(() =>
+            cy
+              .createLabel(secondLabelName, id, {
+                description: 'I hate mieces',
+                color: '#00ff44',
+              })
+              .then(() => {
+                cy.createCell(body.id).then(cell1Resp =>
+                  cy.createMapVariableFromFix('power_vars', id).then(() =>
+                    cy
+                      .createView(body.id, cell1Resp.body.id, 'wumpusDurView')
+                      .then(() =>
+                        cy
+                          .createCell(body.id, {
+                            x: 4,
+                            y: 0,
+                            height: 8,
+                            width: 4,
+                          })
+                          .then(cell2Resp =>
+                            cy
+                              .createView(
+                                body.id,
+                                cell2Resp.body.id,
+                                'sampleNote'
+                              )
+                              .then(() => {
+                                cy.fixture('routes').then(({orgs}) =>
+                                  cy
+                                    .get<Organization>('@org')
+                                    .then(({id}: Organization) => {
+                                      cy.wait(1000)
```

Comment:
any way we could avoid using a `cy.wait(1000)` here? If the page isn't loading fast enough, you can add a `timeout` argument to `cy.get` to make it wait longer. The difference being that cy.wait will always make the test last 1 second longer. But if the page load times improve in the future, increasing the `timeout` only lengthens the test if necessary

Reply:
wait moved to timeout